### ISSUE TITLE
Day04 ex09 : incoherences in the subject

### ISSUE THREAD

**Issue by stelon77** [42-AI/bootcamp_machine-learning#243]
* Day: 04
* Exercise: 09

For part 2 we are asked to have only a train set and a test set and to train from scratch **only** the best model.
In the first part , we have a train, a cross validation and a test set. Our F1 score are calculated from the cross validation set.

Part 2 then asks to print all the F1 scores calculated from test sets. As the sets are different from part 1, we need to train from scratch all the models to do that...

I think you should remove 
For the second script solar_system_census.py, only a train and test
set are necessary as one is simply looking to the performance.

or put back the old version of the subject : asking to print the f1 score of the cross validation set and only the best model on test set

## Fixed on:
- [ ] Github
- [ ] Gitlab
